content/c-sharp/concepts/math-functions/terms/exp/exp.md

```diff
@@ -17,9 +17,7 @@ The **`Math.Exp()`** method returns the value of raising `e` to the power of a g
 
 `e` stands for Euler's number, an irrational number, with an approximate constant value of 2.71828. It provides a constant base to the natural logarithm or log and can be understood as a limit of a given progression.
 
-$$
-\sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots
-$$
+![eFormula](https://user-images.githubusercontent.com/72105079/219855529-5cb30211-ab0d-45ca-bd19-25c36386f83c.jpg)
```

Comment:
The image file needs to be hosted in the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
Use the following url to link to it: https://raw.githubusercontent.com/Codecademy/docs/main/media/image-name.jpg.
Change the name of the image to something like "e-formula.jpg"